gctree/cli.py

```diff
@@ -599,15 +542,16 @@ def get_parser():
         help=r'name of root sequence (outgroup root), default ``"root"``',
     )
     parser_infer.add_argument(
-        "phylipfile",
-        type=str,
-        help="dnapars outfile (verbose output with sequences at each site)",
-    )
-    parser_infer.add_argument(
-        "abundance_file",
+        "infiles",
         type=str,
-        help="File containing allele frequencies (sequence counts) in the "
-        'format: "SeqID,Nobs"',
+        nargs="+",
+        help=(
+            "Input files for inference. If two filenames are passed, the first shall be a "
+            "dnapars outfile (verbose output with sequences at each site), and the second "
+            "shall be an abundance file containing allele frequencies (sequence counts) in "
+            "the format: 'SeqID, Nobs'. If a single filename is passed, it shall be the name "
+            "of a pickled history DAG object created by gctree."
+        ),
     )
     parser_infer.add_argument(
         "--bootstrap_phylipfile",
```

Comment:
Is this argument still supported?

Reply:
No. I will add a warning that it's not, as we already discussed 🙄. Thanks for the reminder

Reply:
I've left it in, and it raises an informative error when passed. Maybe it should just be removed, though?

Reply:
We could just nuke all bootstrap stuff. This reminds me, the old scons pipeline should probably be nuked (as was forewarned in a previous release, since we have a CLI now)—it's probably very broken by now.

Reply:
Sounds fun. I'll do that.